web/public/js/directives/cd-email-users/index.js

```diff
@@ -0,0 +1,70 @@
+;(function () {
+  /*global mailForm*/
+  'use strict';
+
+  angular
+    .module('cpZenPlatform')
+    .component('cdEmailUsers', {
+      bindings: {
+        users: '<',
+        dojoId: '<'
+      },
+      controller: ['usSpinnerService', '$translate', '$rootScope', 'utilsService',
+      'cdDojoService', 'alertService', 'atomicNotifyService', '$scope',
+      function (usSpinnerService, $translate, $rootScope, utilsService,
+        cdDojoService, alertService, atomicNotifyService, $scope) {
+        var ctrl = this;
+        ctrl.editorOptions = utilsService.getCKEditorConfig();
+        var defaultContent = ctrl.content = $translate.instant('Type the content of your email here');
+        ctrl.title = '';
+        cdDojoService.load(ctrl.dojoId)
```

Comment:
Just for code organisation, perhaps we should be put any setup code into `ctrl.$onInit`?